conans/client/graph/install_graph.py

```diff
@@ -285,10 +285,8 @@ def _raise_missing(missing):
         else:
             build_str = " ".join(list(sorted(["--build=%s" % str(pref.ref) for pref in missing_prefs])))
 
-        raise ConanException(textwrap.dedent('''\
-           Missing prebuilt package for '%s'
-           Use 'conan list packages %s --format=html -r=remote > table.html' and open the table.html file to see available packages
-           Or try to build locally from sources with '%s'
-
-           More Info at 'https://docs.conan.io/en/latest/faq/troubleshooting.html#error-missing-prebuilt-package'
-           ''' % (missing_pkgs, ref, build_str)))
+        raise ConanException(textwrap.dedent(f'''\
+           Missing prebuilt package for '{missing_pkgs}'
+           Check the available packages using the 'conan list' command
+           or try to build locally from sources using the '{build_str}' argument
```

Comment:
This change is what is most likely breaking a few tests.
- Are we sure we want to run without the link to the FAQ?
- Shouldn't we be explicit and add ``conan list pkg/version:* --format=html``?

Reply:
why not run command here internally and print the output straight?

Reply:
I added: https://github.com/conan-io/docs/pull/2972 will link to it in the error message